scripts/core/driver.yml

```diff
@@ -271,3 +271,23 @@ members:
     - type: $x_driver_handle_t
       name: handle
       desc: "[in] handle of the driver instance"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Retrieves a message associated with the returned code of the last call made."
+version: "1.6"
+class: $xDriver
+name: GetLastResultString
+details:
+    - "String returned is thread local."
+    - |
+       String may be empty if last call returned $X_RESULT_SUCCESS or if driver
+       determines that error code is already explicit enough to convey error cause.
+    - "Memory pointed by ppString is owned by driver."
+    - "String returned is null-terminated."
+params:
+    - type: $x_driver_handle_t
+      name: hDriver
+      desc: "[in] handle of the driver instance"
+    - type: "const char**"
+      name: ppString
+      desc: "[in,out] pointer to an array of characters containing message associated with the last call."
```

Comment:
Is this array of characters null-terminated? If yes, we should explicitly state that here.

Reply:
it is mentioned in the details, but we can add it here as well of course.

Reply:
Added.